kfdef/kfctl_openshift.yaml

```diff
@@ -0,0 +1,19 @@
+
+apiVersion: kfdef.apps.kubeflow.org/v1beta1
+kind: KfDef
+metadata:
+  name: opendatahub
+  namespace: opendatahub
+spec:
+  applications:
+  - kustomizeConfig:
+      repoRef:
+        name: odh-manifests
+        path: ai-library
+    name: ai-library    
+  repos:
+  - name: manifests
+    uri: https://github.com/opendatahub-io/manifests/tarball/v0.7-branch-openshift
+  - name: odh-manifests
+    uri: https://github.com/opendatahub-io/odh-manifests/tarball/v0.7-branch-openshift
+  version: v0.7-branch-openshift
```

Comment:
did we decide to use the 0.7 branch or 1.0 Kubeflow?

Reply:
Not sure we formalized our final answer on that.  Happy to change this if needed.